WBooks/Model/Comment.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Comment.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 26/08/21.
+//
+
+import Foundation
+
+internal struct Comment: Codable {
+    
+    let id: Int
+    let user_id: Int
+    let book_id: Int
+    let content: String
+    
+    enum Comment: String, CodingKey {
```

Comment:
If the var name and the string name is the same, we can avoid it. 
Coding keys is useful when the api name fields is in Spanish or if it are in snake case and we use the var name as camel case.

For example, we can change this like:

enum Comment: String, CodingKey {
        case id
        case userId = "user_id"
        case bookId = "book_id"
        case content

Reply:
Done

Reply:
Done